src/index.md

```diff
@@ -121,13 +123,12 @@ Service Classes and their corresponding Service Level Objectives are defined in
 
   ```js
   resize((width) => Plot.plot(addPlotCaption({
-    title: "\"Warm\" Service Class Conformance",
-    subtitle: html`Providers conforming to warm service class`,
+    title: "\"Warm\" Service Class Conformance: Retrievability",
+    subtitle: html`Providers meeting the "Warm" service class retrievability SLO`,
     x: { label: "Date" },
     y: {
       grid: true,
-      label: "Meet Retrieval SLI",
-      domain: [0, 100]
```

Comment:
I assume we don't want a fixed domain since this will grow based on the number of SPs who conform.

Reply:
Yes! This was a leftover from when I started with percentages.